R/temporal_slice.R

```diff
@@ -9,9 +9,6 @@
 #' @title TemporalSlice
 #' @description TemporalSlice Class
 #' @format An \code{R6Class} generator object
-#' @field item  \link{object} [optional]
-#'
-#' @field value  \link{object} [optional]
```

Comment:
Eh?

Reply:
I was getting warnings that these objects were not linked to anything

Reply:
The CRAN compiler thing didn't like the \link for some reason

Reply:
Let's just leave them out then?

Reply:
I'm not sure, it would apply to all the \links above. I can take another look and see if I can figure out why there was an issue so we don't have to remove them